salt/renderers/pass.py

```diff
@@ -75,18 +99,66 @@ def _fetch_secret(pass_path):
     Fetch secret from pass based on pass_path. If there is
     any error, return back the original pass_path value
     """
-    cmd = "pass show {}".format(pass_path.strip())
-    log.debug("Fetching secret: %s", cmd)
+    pass_exec = _get_pass_exec()
+
+    # Make a backup in case we want to return the original value without stripped whitespaces
+    original_pass_path = pass_path
+
+    # Remove the optional prefix from pass path
+    pass_prefix = __opts__.get("pass_variable_prefix", None)
+    if pass_prefix:
+        # If we do not see our prefix we do not want to process this variable
+        if not pass_path.startswith(pass_prefix):
```

Comment:
Can you make sure we document for users that if the the pass path doesn't start with the prefix the renderer will return the path.

Reply:
I've rephrased the documentation to make this more clear.